modules/afsql/afsql.c

```diff
@@ -384,15 +384,15 @@ afsql_dd_create_index(AFSqlDestDriver *self, const gchar *table, const gchar *co
   if (strcmp(self->type, s_oracle) == 0)
     {
       /* NOTE: oracle index indentifier length is max 30 characters
-       * so we use the first 30 characters of the table_column md5 hash */
+       * so we use the first 30 characters of the table_column SHA hash */
       if ((strlen(table) + strlen(column)) > 25)
         {
 
-          guchar hash[MD5_DIGEST_LENGTH];
+          guchar hash[SHA_DIGEST_LENGTH];
           gchar hash_str[31];
           gchar *cat = g_strjoin("_", table, column, NULL);
 
-          MD5((guchar *)cat, strlen(cat), hash);
+          SHA1((guchar *)cat, strlen(cat), hash);
```

Comment:
We've discussed this IRL and we decided to keep the old MD5 index to be fully compatible with older versions.

A non-deprecated API can be used: EVP_Digest.